src/main/conversions.c

```diff
@@ -2382,7 +2382,15 @@ as_status get_cdt_ctx(AerospikeClient *self, as_error *err, as_cdt_ctx *cdt_ctx,
                       PyObject *op_dict, bool *ctx_in_use,
                       as_static_pool *static_pool, int serializer_type)
 {
-    PyObject *py_ctx = PyDict_GetItemString(op_dict, CTX_KEY);
```

Comment:
I think it might be better to modify Client.index_cdt_create() and Query.where() directly, instead of this helper function. Many other functions use this helper function, so this change might introduce a bug somewhere else in the client.

Reply:
You are right.  I added these changes in the [newest commit](https://github.com/aerospike/aerospike-client-python/pull/550/commits/1c7d6c8a21d8bdc5d4891337b62000b3bf1a031f)